src/main/resources/feedbackQuestionConstSumEditFormOptionFragment.html

```diff
@@ -1,5 +1,5 @@
-<div id="constSumOptionRow-${i}-${questionNumber}">
-    <div class="input-group col-sm-12">
+<div class="margin-bottom-7px" id="constSumOptionRow-${i}-${questionNumber}">
```

Comment:
I think it would be better if we dont have the `7px` in the name. cos if we have to change that value for some reason, it would result in a huge changeset. rather if it is used a lot for the questions, name it as `margin-bottom-question-offset` or smth